WordPress/Classes/ViewRelated/NUX/LoginViewController.m

```diff
@@ -821,10 +632,13 @@ - (void)showCreateAccountView
     [self.navigationController pushViewController:createAccountViewController animated:YES];
 }
 
-- (void)showJetpackAuthentication
+- (void)showJetpackAuthentication:(NSNumber *)blogId
```

Comment:
A couple things:
- I'd rename this to `showJetpackAuthenticationForBlog:`
- I understand we don't want to be passing managed objects around, but how about passing the `NSManagedObjectID` for performance? Actually, screw performance, I don't think we're storing `blogID` for self hosted, and if we did most of them would be `1` anyway